tests/apps/courses/test_organization_plugin.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+"""
+Unit tests for the Organization plugin and its model
+"""
+from django import forms
+from django.conf import settings
+from django.test import TestCase
+
+from cms.api import add_plugin, create_page
+from cmsplugin_plain_text.cms_plugins import PlaintextPlugin
+from djangocms_picture.cms_plugins import PicturePlugin
+
+from richie.apps.core.factories import FilerImageFactory, UserFactory
+from richie.apps.core.helpers import create_i18n_page
+from richie.apps.courses.cms_plugins import OrganizationPlugin
+from richie.apps.courses.factories import OrganizationFactory
+from richie.apps.courses.models import OrganizationPluginModel
+
+
+class OrganizationPluginTestCase(TestCase):
+    """
+    Test that OrganizationPlugin correctly displays a Organization's page placeholders content
+    """
+
+    def test_organization_plugin_form_page_choices(self):
+        """
+        The form to create a organization plugin should only list organization pages in the select box.
+        """
+
+        class OrganizationPluginModelForm(forms.ModelForm):
+            """A form for testing the choices in the select box"""
+
+            class Meta:
+                model = OrganizationPluginModel
+                exclude = ()
+
+        organization = OrganizationFactory()
+        other_page_title = "other page"
+        create_page(other_page_title, "richie/fullwidth.html", settings.LANGUAGE_CODE)
+        plugin_form = OrganizationPluginModelForm()
+        self.assertIn(organization.extended_object.get_title(), plugin_form.as_table())
+        self.assertNotIn(other_page_title, plugin_form.as_table())
+
+    def test_organization_plugin_render(self):
+        """
+        Test that a OrganizationPlugin correctly renders organization's page specific information
+        """
+        # Create a filer fake image
+        staff = UserFactory(is_staff=True, is_superuser=True)
+        image = FilerImageFactory(owner=staff)
+
+        # Create a Organization
+        organization = OrganizationFactory(title="La Sorbonne", logo="my_logo.jpg")
+        organization_page = organization.extended_object
```

Comment:
you can ask a title object in another language because your section below "the same tests in French" is not really done ;-)

Reply:
oups, i didn't notice i was missing some code from copying the person test. Added some assertContains for the french portion.